e2e/suites/application/page-titles.test.ts

```diff
@@ -81,8 +81,10 @@ describe('Page titles', () => {
     });
 
     it('File Libraries page - [C217158]', async () => {
+      const parent = SIDEBAR_LABELS.FILE_LIBRARIES;
       const label = SIDEBAR_LABELS.FILE_LIBRARIES;
```

Comment:
should "label" be in fact SIDEBAR_LABELS.MY_LIBRARIES ?

Reply:
do we have different page titles for My Libraries and Favorite Libraries ?

Reply:
yes